Doc/library/argparse.rst

```diff
@@ -2400,3 +2080,15 @@ Exceptions
 .. exception:: ArgumentTypeError
 
    Raised when something goes wrong converting a command line string to a type.
+
+.. We use the "rubric" directive here to avoid creating
```

Comment:
I copied this from the asyncio docs. There might be a better way to do this?

Reply:
On asyncio the "Reference" section points to the reference docs for individual modules within the asyncio package, but that's not the case here, so I don't think the name "Reference" works here. Maybe use "Guides and Tutorials" as a normal heading (included in the TOC) and remove the `:caption:`.